en/_posts/2024-12-05-groonga-14.1.1.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post.en
+title: groonga 14.1.1 has been released
+description: groonga 14.1.1 has been released!
+published: false
```

Comment:
The current setting is false so we won't publish this page even if we merge this PR.